### PR TITLE
MINOR: Add log message when deleting topics

### PR DESCRIPTION
**Pull request by stanislavkozlovski**
This patch adds a log message that indicates what topics are attempting deletion

### CODE REVIEW

**Comment by stanislavkozlovski** [comment]
Apparently, we don't log in INFO when deleting topics

We log when creating topics https://github.com/apache/kafka/blob/trunk/core/src/main/scala/kafka/zk/AdminZkClient.scala#L88, so it's only fair we log when we delete them as well :)

**Comment by stanislavkozlovski** [comment]
Turns out this exists. My bad - https://github.com/apache/kafka/blob/trunk/core/src/main/scala/kafka/controller/KafkaController.scala#L1409

**Comment by stanislavkozlovski** [comment]
re-opening -- we log when deleting topics on the controller but ~other brokers can receive the request and write to ZK as well~. I believe it might be useful to log in the API layer, just like with topic creation

**Comment by stanislavkozlovski** [comment]
cc @hachikuji (added the log line for topic creation) @gwenshap @ijuma @omkreddy for reviews

**Comment by ijuma** [comment]
Topic deletion should go to the controller right?

**Comment by stanislavkozlovski** [comment]
Right. I had misread the code, sorry about that.
I still think it might be a useful log

**Comment by stanislavkozlovski** [comment]
retest this please

**Comment by stanislavkozlovski** [comment]
Rebased with master.
Any thoughts on this? cc @omkreddy @ijuma @gwenshap

File: core/src/main/scala/kafka/server/AdminManager.scala [hunk 1]
```diff
@@ -191,7 +191,7 @@ class AdminManager(val config: KafkaConfig,
   def deleteTopics(timeout: Int,
                    topics: Set[String],
                    responseCallback: Map[String, Errors] => Unit): Unit = {
-
+    info(s"Deleting topics ${topics.mkString(", ")}")
```
Comment (ijuma) [inline core/src/main/scala/kafka/server/AdminManager.scala:194]: The other methods in this class don't seem to have a log line like this one. Seems like we should be consistent one way or another. Or do we think deletion is special in some way?

Comment (stanislavkozlovski) [reply core/src/main/scala/kafka/server/AdminManager.scala:194]: I think that we should have logs for operations that update resources. Adding logs to `createPartitions()` and `createTopics()` sounds good to me, as long as they aren't overly verbose.

For incrementalAlterConfigs/alterConfigs, I'm not sure

**Pull request closed without merging**